.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
     - env: PYTHON=3.4 QT_VERSION=4 IPY_VERSION=3
     - env: PYTHON=3.5 QT_VERSION=5 IPY_VERSION=5 QSCINTILLA=0
     - env: PYTHON=3.6 QT_VERSION=5 IPY_VERSION=6 QSCINTILLA=1
+    - env: PYTHON=3.7 QT_VERSION=5 IPY_VERSION=6 QSCINTILLA=1
```

Comment:
I am not sure we can claim support just yet. In particular I suspect that fast call related changes may need some special care.

Reply:
But we can keep it in.

Reply:
I was going to create a ticket to track support for 3.7, but I tested my applications and didn't have any issues.   

I dissembled some code and  it shows the new `LOAD_METHOD` and `CALL_METHOD` are used. 

Ex

```python

Python 3.7.0 (default, Jun 28 2018, 13:15:42) 
Type 'copyright', 'credits' or 'license' for more information
IPython 6.4.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import dis

In [2]: from atom.api import *
   ...: class Test(Atom):
   ...:     x = Unicode()
   ...:     def bar(self):
   ...:         return self.foo()
   ...:     def foo(self):
   ...:         return self.x
   ...:     

In [3]: t = Test()

In [4]: dis.dis(t.bar)
  5           0 LOAD_FAST                0 (self)
              2 LOAD_METHOD              0 (foo)
              4 CALL_METHOD              0
              6 RETURN_VALUE

In [5]: dis.dis(t.foo)
  7           0 LOAD_FAST                0 (self)
              2 LOAD_ATTR                0 (x)
              4 RETURN_VALUE

In [6]: def baz():
   ...:     t.foo()
   ...:     

In [7]: dis.dis(baz)
  2           0 LOAD_GLOBAL              0 (t)
              2 LOAD_METHOD              1 (foo)
              4 CALL_METHOD              0
              6 POP_TOP
              8 LOAD_CONST               0 (None)
             10 RETURN_VALUE

```

Will the tracer need updated to handle these?

Reply:
Yes I suspect so but I need to dig a bit more before being sure.